### ISSUE TITLE
Can't update to 1.3.x

### ISSUE THREAD

**Issue by mark-adigital** [verbb/formie#267]
Hi - I've been trying to update Formie on a dev site, but can't seem to get past 1.2.28. Attempting to do so results in an invalid configuration error: Unable to locate message source for category 'formie'.

<img width="1368" alt="Screenshot 2020-12-16 at 14 37 43" src="https://user-images.githubusercontent.com/30312669/102373793-7b4b5900-3fb8-11eb-8c33-20babcacd0dd.png">

I've tried this back on Craft 3.4.x and on the latest release (3.5.17) and have seen the same error. It seemingly relates to translations: https://craftcms.stackexchange.com/questions/11065/unable-to-locate-message-source-for-category

Thanks!

**Comment by engram-design** [comment]
I have a feeling this might be related to https://github.com/verbb/formie/issues/179 where the agree field has a setting for `descriptionHtml` which it _should_ be taking care of.

Can you confirm by checking for any `verbb\formie\fields\formfields\Agree` fields in your `fields` database table, and removing `descriptionHtml` in the settings column?

**Comment by mark-adigital** [comment]
I've checked that table and there's no fields of that type at all, and none with `descriptionHtml` in the settings column either.

**Comment by engram-design** [comment]
Wondering if you can try out the latest code in the craft-3 branch to get rid of this error. It's actually trying to save the error to a log. Change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mark-adigital** [comment]
Yep, can confirm that has worked for me.

**Comment by engram-design** [comment]
Fixed in [1.3.5](https://github.com/verbb/formie/releases/tag/1.3.5)
